### PR TITLE
LPS-193416 lack of user on PrincipalThreadLocal when deleting company on test

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
https://liferay.atlassian.net/browse/LPS-193416

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/adaptive-media/adaptive-media-image-test/src/testIntegration/java/com/liferay/adaptive/media/image/service/impl/test/AMImageEntryLocalServiceTest.java [hunk 1]
```diff
@@ -426,13 +431,16 @@ public void testGetExpectedAMImageEntriesCount() throws Exception {
 			_unregisterAMImageCounter(amImageCounterServiceRegistration);
 
 			_companyLocalService.deleteCompany(company);
+			PrincipalThreadLocal.setName(originalName);
```
Comment (adolfopa) [inline modules/apps/adaptive-media/adaptive-media-image-test/src/testIntegration/java/com/liferay/adaptive/media/image/service/impl/test/AMImageEntryLocalServiceTest.java:434]: Don't we need a blank line between `_companyLocalService` and `PrincipalThreadLocal`? Same in other places.

File: modules/apps/adaptive-media/adaptive-media-image-test/src/testIntegration/java/com/liferay/adaptive/media/image/service/impl/test/AMImageEntryLocalServiceTest.java [hunk 2]
```diff
@@ -530,6 +547,7 @@ public void testGetPercentageMax100() throws Exception {
 			null;
 
 		try {
+			PrincipalThreadLocal.setName(user.getUserId());
```
Comment (adolfopa) [inline modules/apps/adaptive-media/adaptive-media-image-test/src/testIntegration/java/com/liferay/adaptive/media/image/service/impl/test/AMImageEntryLocalServiceTest.java:550]: I think a blank line is required after `PrincipalThreadLocal`.

**Review by adolfopa — CHANGES_REQUESTED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**